src/main/java/org/sonarlint/intellij/messages/GlobalConfigurationListener.java

```diff
@@ -21,11 +21,32 @@
 
 import com.intellij.util.messages.Topic;
 import java.util.List;
+import java.util.function.BiConsumer;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.function.BiConsumer'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)